terraform-security.md

```diff
@@ -0,0 +1,19 @@
+# Terraform Security
+
+1. Are you storing your Terraform Code in a version control system (Git or similar)?
+1. Are you using one Terraform workspace for each environment of a given infrastructure component?
+1. Are you using Terraform modules to increase reusability of your infrastructure code?
+1. How do you handle sensitive information in your Terraform scripts?
+    1. How are the credentials of a Terraform service account (JSON) secured?
+    1. Where are these credentials stored?
+    1. Is your Terraform integrated with a secret management tool (HashiCorp’s Vault or similar)?
+    1. Is your Terraform state file encrypted at rest?
+1. Are the permissions of a Terraform service account restricted to minimum?
```

Comment:
Because there is no thing as a "service account" concept in Terraform to my knowledge, what do you mean when you write "Terraform service account"?

Reply:
I meant the service account in GCP - A service account is a special type of Google account intended to represent a non-human user that needs to authenticate and be authorized to access data in Google APIs.